openshift-logstash/values.yaml

```diff
@@ -23,6 +23,10 @@ image:
 # logstash env variable
 env:
 #  LS_JAVA_OPTS: '-Xms512m -Xmx512m'  # check resources too!
+  - '--path.data=/tmp'
+  - '-f'
+  - '/etc/logstash/conf/*.conf'
+
```

Comment:
These should be under a "args" key, not under the *env" key.

Reply:
My bad, i was too greedy during the conflict resolution :laughing: